AUTHORS

```diff
@@ -1,3 +1,3 @@
 Nicholas H.Tollervey (ntoll@ntoll.org)
 Matt Wheeler (m@funkyhat.org)
-
+Edmond Locard (L0C4RD@gmail.com)
```

Comment:
LinkedIn: https://www.linkedin.com/in/ed-locard-a50740127
Twitter: https://twitter.com/L0C4RD
Talk I did recently (think my name is on one of the slides): https://youtu.be/0JU5DofALik

I hope it's understandable that I try to minimise my online presence - the work that I do is quite often misrepresented in the media, and on more than one occasion I've had people make threats to me, both across the internet and in person. Unfortunately, having a weird name makes it that much easier to find me.

Because of all this, I don't get as many opportunities to work on amazing community projects as I'd like. However after reading your contributions guidelines [here](https://github.com/ntoll/uflash/blob/master/CONTRIBUTING.rst), and being really impressed by what seemed to be a radically inclusive outlook on contributions, I thought you guys might be willing to accept some of my work, even lacking, as I am, a huge public presence - please prove me right!

pokes (as it's been a little while) : @ntoll